nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/client/ESRestClientFactory.java

```diff
@@ -214,7 +214,7 @@ protected SSLContext getSslContext(ElasticSearchClientConfig config) {
                 sslContextBuilder.loadTrustMaterial(trustStore, null);
             }
             if (keyStore != null) {
-                sslContextBuilder.loadKeyMaterial(keyStore, null);
+                sslContextBuilder.loadKeyMaterial(keyStore, keyStorePassword.toCharArray());
```

Comment:
Please do the same as #3333 to be safe:

                sslContextBuilder.loadKeyMaterial(keyStore,
                         StringUtils.isBlank(keyStorePassword) ? null : keyStorePassword.toCharArray());

Reply:
yes of course!